youtube_dl/extractor/beatport.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class BeatportIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?beatport\.com/(?P<uploader_id>\w+)/tracks/(?P<id>.+)/(?P<display_id>.+)'
+    _TESTS = [{
+        'url': 'https://www.beatport.com/musicalfreedom/tracks/l2axnqcd6azu/secrets-feat-vassy-original-mix',
+        'md5': '11e330f6236be07f61fcc4669e972cd8',
+        'info_dict': {
+            'id': 'l2axnqcd6azu',
+            'display_id': 'secrets-feat-vassy-original-mix',
+
+            'uploader': 'Musical Freedom',
+            'uploader_id': 'musicalfreedom',
+            'uploader_url': 'http://www.musicalfreedom.com',
+
+            'ext': 'mp4',
+            'title': 'Secrets feat. Vassy',
+            'duration': 267441,
+            'artist': ['Tiësto', 'VASSY', 'KSHMR'],
```

Comment:
[![artist_list](https://cloud.githubusercontent.com/assets/4989736/15093030/2c923526-147b-11e6-96eb-90b61c7e3e01.png)](https://www.beatport.com/musicalfreedom/tracks/l2axnqcd6azu/secrets-feat-vassy-original-mix)
Okey, i will add a space to the seperator.